tools/cellprofiler/starting_modules.xml

```diff
@@ -1,5 +1,5 @@
-<tool id="cp_common" name="cellProfiler common start modules" version="@CP_VERSION@">
-    <description>load images and metadata into cellProfiler</description>
+<tool id="cp_common" name="Starting modules" version="@CP_VERSION@">
+    <description> (Images, Metadata, NamesAndTypes, Groups) load images and metadata into CellProfiler</description>

```

Comment:
the bracket at the end?

Reply:
did you mean here that the description should be shorter?

Reply:
No, just "<description>load images and metadata into CellProfiler (Images, Metadata, NamesAndTypes, Groups)</description>"